services/wireguard/endpoint/userspace/network_windows.go

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package userspace
+
+import (
+	"net"
+	"os/exec"
+	"strconv"
+
+	"github.com/jackpal/gateway"
+	"github.com/pkg/errors"
+)
+
+func assignIP(iface string, subnet net.IPNet) error {
+	out, err := exec.Command("powershell", "-Command", "netsh interface ip set address name=\""+iface+"\" source=static "+subnet.String()).CombinedOutput()
```

Comment:
this powershell will definitely will not be available on pre Win7 versions. We probably need to state that somewhere in docs.. and requirements.

Reply:
Pre Win7 versions of windows officially not supported, I don't think that we should worry about it.